components/Header/index.js

```diff
@@ -1,3 +1,17 @@
+import styled from "styled-components";
+
 export default function Header() {
-  return <h1>TrailBlazer</h1>;
+  return <StyledHeader>TrailBlazer</StyledHeader>;
 }
+
+const StyledHeader = styled.h1`
+  background-color: var(--dijon);
+  padding: 10px 0;
+  margin: 0;
+  border: 1px solid grey;
+  position: fixed;
+  width: 100vw;
+  z-index: 3;
+  box-shadow: rgba(0, 0, 0, 0.4) 0px 2px 4px,
```

Comment:
You're working on a 'global styles' userstory, right? 
What about defining shadow variables instead of defining the same shadow at different places?